### ISSUE TITLE
input "attestations" is not defined in action "pypa/gh-action-pypi-publish@release/v1"

### ISSUE THREAD

**Issue by hugovk** [rhysd/actionlint#445]
Add `attestations: true` to a `pypa/gh-action-pypi-publish` step:

```yaml
    steps:
      - name: Upload package to PyPI
        uses: pypa/gh-action-pypi-publish@release/v1
        with:
          attestations: true
```

For example: 
* https://github.com/hugovk/tinytext/commit/7b0ac9ad7ed7d8b2f08dc393cd813c7446c49853
* https://github.com/hugovk/tinytext/actions/runs/10662143949/job/29548819273

Fails with:

```
.github/workflows/deploy.yml:93:11: input "attestations" is not defined in action "pypa/gh-action-pypi-publish@release/v1". available inputs are "packages-dir", "packages_dir", "password", "print-hash", "print_hash", "repository-url", "repository_url", "skip-existing", "skip_existing", "user", "verbose", "verify-metadata", "verify_metadata" [action]
   |
93 |           attestations: true
   |           ^~~~~~~~~~~~~
```

This was recently added to the action:

* https://github.com/pypa/gh-action-pypi-publish#generating-and-uploading-attestations
* https://github.com/pypa/gh-action-pypi-publish/releases/tag/v1.10.0
* https://github.com/pypa/gh-action-pypi-publish/commit/8a08d616893759ef8e1aa1f2785787c0b97e20d6

Does this need adding to schema store or something?

**Comment by maikelvdh** [comment]
This should be addressed with https://github.com/rhysd/actionlint/pull/446

**Comment by maikelvdh** [comment]
This should be addressed by https://github.com/rhysd/actionlint/pull/446

**Comment by rhysd** [comment]
This issue should be fixed by [v1.7.2](https://github.com/rhysd/actionlint/releases/tag/v1.7.2).

**Comment by hugovk** [comment]
Confirmed, thank you!
